dss-core/src/main/java/io/github/ztkmkoo/dss/core/network/rest/enumeration/DssLogLevel.java

```diff
@@ -0,0 +1,23 @@
+package io.github.ztkmkoo.dss.core.network.rest.enumeration;
+import io.netty.handler.logging.LogLevel;
+import lombok.Getter;
+
+public enum DssLogLevel {
+
+    ERROR(LogLevel.ERROR), WARN(LogLevel.WARN), INFO( LogLevel.INFO), DEBUG( LogLevel.DEBUG), TRACE( LogLevel.TRACE);
```

Comment:
I appreciate you reformatting this enumeration elements, each element in each line (not put all the elements in one line)